scripts/git_page/md_post_process.py

```diff
@@ -0,0 +1,191 @@
+import argparse
+from pathlib import Path
+import logging
+from typing import Union
+
+import utility as util
+
+Logger = logging.getLogger(__name__)
+logging.basicConfig()
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Performs post processing on markdown files. - Adds title header to markdown files if it does not "
+    )
+    parser.add_argument(
+        "--md_dir", metavar="MARKDOWN_DIRECTORY", required=True,
+        help="One of or more directories containing markdown "
+             "documentation files for workflow"
+    )
+    parser.add_argument("--debug", action="store_true", help="verbose logging")
+
+    args = parser.parse_args()
+
+    util.set_logging_level(args)
+
+    md_dir_name = args.md_dir
+
+    md_dir = util.get_absolute_path(md_dir_name)
```

Comment:
It looks like `md_dir_name` is a single dir, not the multiple folders as indicated in the doc?

Reply:
done